v8/mastodoncomments/mastodoncomments.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2021 Pieter David
```

Comment:
You probably want your name here :)

Reply:
Yeah, everybody should know who to blame. :-D